src/config/entities/__tests__/configuration.ts

```diff
@@ -16,18 +17,21 @@ export default (): ReturnType<typeof configuration> => ({
     apiKey: faker.string.hexadecimal({ length: 32 }),
     cacheTtlSeconds: faker.number.int(),
   },
-  safeConfig: {
-    baseUri: faker.internet.url({ appendSlash: false }),
+  expirationTimeInSeconds: {
+    default: faker.number.int(),
   },
-  safeTransaction: {
-    useVpcUrl: false,
+  httpClient: { requestTimeout: faker.number.int() },
+  log: {
+    level: sample(['error', 'warn', 'info', 'debug']) ?? 'debug',
   },
   redis: {
     host: faker.internet.domainName(),
     port: faker.internet.port().toString(),
   },
-  expirationTimeInSeconds: {
-    default: faker.number.int(),
+  safeConfig: {
+    baseUri: faker.internet.url({ appendSlash: false }),
+  },
+  safeTransaction: {
+    useVpcUrl: false,
   },
```

Comment:
Just ordering alphabetically here (besides adding the `log` property`).

Reply:
Can you cherry pick this to a separate PR? Rolling back the `log` should not roll back this sorting in my opinion.

Reply:
Done! https://github.com/safe-global/safe-client-gateway-nest/pull/519